docs/developer-guide/rosplane/estimator/estimator-overview.md

```diff
@@ -0,0 +1,95 @@
+This page will outline the states, their meaning and any general notes on the states.
+For a more in depth look at which states are estimated in which way visit the [Estimator Base](./estimator-base.md) and [Estimator Example](./estimator-example.md) pages.
+
+## ROS Interactions
+
+The estimator takes in sensor information from `rosflight_io` computes an estimate and publishes it to the rest of ROSplane.
+
+
+| ![Diagram of Estimator ROS Interactions](../../../assets/estimator_assets/estimator_ros_input_output.png "Diagram of Estimator ROS Interactions") |
+|:--:|
+|*Figure 1: ROS network interactions for the estimator.*|
+
+### Input
+
+The inputs to the estimator are, accelerometer, rate gyro, barometer, differential pressure, GPS position, and GPS velocity estimates.
+The table with the topic for each of the measures is below.
+
+| Measure | Explanation | Topic |
+|:------:|:-------:| :---: |
+| Accelerometer | This measures the specfic force applied to the aircraft in the body frame axes (see section 7.1 in the UAV book for more details). | `/imu/data/linear_acceleration` |
+| Rate Gyro | This measures the angular velocity of the aircraft around the body frame axes. | `/imu/data/angular_velocity` |
+| Barometer | The barometer measures the ambient air pressure. It is calibrated on arm to establish a "zero" altitude measurement. | `/baro/pressure` |
+| Differential Pressure | The differential pressure sensor, measures the difference in pressure using a pitot tube due to forward velocity. | `/airspeed/differential_pressure` |
+| GNSS Position | GNSS postion gives the position of the aircraft in latitude, longitude and altitude. | `/navsat_compat/fix` |
+| GNSS Velocity | GNSS velocity gives the velocity of the aircraft in meters per second in the global NED frame. | `/navsat_compat/vel` |
+| Status | Indicates whether the aircraft is armed (indicating a need to initialize position and altitude estimates). | `/status` |
+
+These topics provide the measures that are fused to create a state estimate.
+
+| ![Diagram of Estimator ROS IO](../../../assets/estimator_assets/estimator_ros_input_output.png "Diagram of Estimator ROS IO") |
+|:--:|
+|*Figure 1: ROS topic subscriptions and publications for the estimator.*|
+
+## Output
+
+There are 20 states estimated by the estimator that are published to the rest of ROSplane.
+These states cover the position, orientation and aerodynamic information for the aircraft.
+Below is a table of the `/estimated_state` message and what each of the fields represents.
+Note that there are more than 20 states listed in the following table, but this is because either Euler angles or quaternions can be used to express orientation.
+See the Frames and Derivation page for more information, or chapter 2 of the [UAV book](https://github.com/randybeard/mavsim_public).
```

Comment:
We should add a link to the Frames and Derivation page. Is the page you're referring to in Ch2 of the UAV book?

Reply:
I was going to link to a page on the website that covers that topic at some point. I think it may be useful to just have a diagram of all the frames and what things are and leave Chapter 2 for explaining the rationale for choosing them.

Reply:
That would be a great page to have.

Reply:
I am going to add it to the misc improvements and resolve this.